pkg/datarepair/repairer/repairer.go

```diff
@@ -140,6 +140,9 @@ func (service *Service) process(ctx context.Context) (err error) {
 
 func (service *Service) worker(ctx context.Context, seg *pb.InjuredSegment) (err error) {
```

Comment:
is this method tested anywhere?

Reply:
The datarepair integration test (`pkg/datarepair/datarepair_test.go`) ends uses this method indirectly